extensions/ql-vscode/package.json

```diff
@@ -757,38 +757,74 @@
         "command": "codeQLDatabases.displayAllLanguages",
         "title": "All languages"
       },
+      {
+        "command": "codeQLDatabases.displayAllLanguagesSelected",
```

Comment:
It is a bit tedious that we have to do it in this way. But maybe we can get a better way later.